config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get '/', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [Optional] It would be better professionally if you add your routes like that:

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```